man/citgm-all.1

```diff
@@ -70,6 +70,12 @@ Run tests in parallel (automatically detect core count)
 .TP
 .BR \-\-tmpDir " " \fI<path>\fR
 Directory to test modules in
+.TP
+.BR \-\-includeTags " " \fI<tag1,tag2>\fR
```

Comment:
Is it the correct syntax? The parameters are not supposed to be comma-separated.